docs/nextsteps.rst

```diff
@@ -0,0 +1,124 @@
+.. _next-steps:
+
+Next Steps
+==========
+
+Setting Up Continuous Integration
+---------------------------------
+
+Once you have rendered your package and set it up on `GitHub
+<https://github.com>`__ you may wish to enable Travis CI, Appveyor and Read the
+Docs (configuration for these services is included in the template, but you
+could utilise others).
+
+Travis CI
+#########
+
+You should register your package on https://travis-ci.org and modify the
+``.travis.yml`` file to your needs. The default ``.travis.yml`` file contains a
+large number of builds, against various versions of Python and Astropy and
+numpy, you should choose the ones relevant to your project. Full documentation
+of the ``.travis.yml`` file can be found on the
+`Travis CI <https://docs.travis-ci.com/user/for-beginners/>`__ website.
+
+Continuous Integration services like Travis CI, continuously test your package
```

Comment:
Maybe put this as the *first* paragraph of this section? since it's justifying why anyone would care about this section?